Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -3650,6 +3670,10 @@ public void reInit() {
             init();
         }
     }
+
+    boolean isIDLSession() {
+        return "true".equals(instrumentationExtraData_.get(Defines.Jsonkey.InstantDeepLinkSession.getKey()));
+    }
```

Comment:
instead of doing a string comparison here, can we do a Boolean parse test?   That way, just in case someone uses "True", "true", or "TRUE" it will still work.